modules/porous_flow/tests/kernels/simple_diffusion/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [./test]
+    type = 'Exodiff'
+    input = 'simple_diffusion.i'
+    exodiff = 'simple_diffusion_out.e'
+  [../]
+[]
```

Comment:
Need newline at EOF.  I guess this needs to be fixed in stork, since it keeps getting propagated from there, presumably.  I'll do that.